### PR TITLE
Deploi potfolio page/live version

### PR DESCRIPTION
**Pull request by Ngufor-emmanuella**
In this project i built a desktop portfolio version webpage. grid and flex position elements are used to position elements in my webpage. Also on buttons hover was applied to them . And a contact form was added to finalise the mobile version portfolio. also a desktop version was created . the page is responsive both on mobile and desktops screens. min width pf 375px for mobile version and a min width of 768px for desktop versions
An animation property was also added in the nav bars in the header section to animate the page.no complex code is done here.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ An animation propert was also added in the nav bars in the header section to ani
 
 ### Install
 
-- Copy the github link for the repository [repository link](https://github.com/Ngufor-emmanuella/personal-portfolio.git)
+- Copy the github link for the repository [repository link]( https://ngufor-emmanuella.github.io/personal-portfolio/)
```
Comment (prolajumokeoni) [inline README.md]: ### OPTIONAL

- [ ] Kindly move the live demo link to the deployment section so users can easily find the live demo link 👍🏿 


- You can change your repository link to link of your project repository 👍🏿

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
## OPTIONAL

**Review by prolajumokeoni — APPROVED** [review]
Hi @Ngufor-emmanuella 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
 - Good design 💯 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Ngufor-emmanuella) [reply README.md]: thanks already did the changes

**Pull request merged**